chaincode/ledger_db.go

```diff
@@ -383,3 +384,73 @@ func (db *LedgerDB) GetNode(key string) (Node, error) {
 
 	return node, nil
 }
+
+// ----------------------------------------------
+// High-level functions for events
+// ----------------------------------------------
+
+// SendTuplesEvent sends an event with updated tuples if there is any
+// Only one event can be sent per transaction
+func (db *LedgerDB) SendTuplesEvent() error {
```

Comment:
I think you forgot to remove the original method from `utils.go`

Reply:
confirmed with @thibaultrobert this needs fixing

Reply:
fixed: https://github.com/SubstraFoundation/substra-chaincode/pull/41/commits/d06ce2fd5387ea3c48400161227abd2f6c1be4d2